src/store/plugins/names.js

```diff
@@ -69,27 +69,31 @@ export default (store) => {
         dispatch,
       }) {
         if (!middleware) return;
-        const getPendingNameClaimTransactions = (address) => dispatch('fetchPendingTransactions', address, { root: true }).then((transactions) => transactions
-          .filter(({ tx: { type } }) => type === 'NameClaimTx')
-          .map(({ tx, ...otherTx }) => ({
-            ...otherTx,
-            ...tx,
-            owner: tx.accountId,
+        const getPendingNameClaimTransactions = (address) => dispatch('fetchPendingTransactions', address, { root: true })
+          .then((transactions) => transactions
+            .filter(({ tx: { type } }) => type === 'NameClaimTx')
+            .map(({ tx, ...otherTx }) => ({
+              ...otherTx,
+              ...tx,
+              owner: tx.accountId,
+            })));
+
+        const getNames = (address) => middleware.getNamesOwnedBy(address)
+          .then(({ active }) => active.map(({ info, name, hash }) => ({
+            createdAtHeight: info.activeFrom,
+            expiresAt: info.expireHeight,
+            owner: info.ownership.current,
+            pointers: info.pointers,
+            nameHash: hash,
```

Comment:
Why did you decide to not just add a `nameHash: hash`, but extract the whole `getNames` function?

Reply:
Only for better code readability and consistency with the second function in `names` assignment